### PR TITLE
PHP 8: Code Review `MediaPool`

### PR DESCRIPTION
**Pull request by mjansenDatabay**
Hi @alex40724 ,

I completed the review of the `Modules/MediaPool` component.

Results:
- [x] The code style is `PSR-2 + X` compliant
- [x] No usage of `$_GET` / `$_POST` / `$_REQUEST` / `$_SESSION`
- [x] There is a `Unit Test Suite` with at least one unit test
- [x] The unit tests pass
- [x] External libraries are compatible with PHP 8 (if relevant)
- [ ] There are no serious `PhpStorm Code Inspection` issues left
- [ ] The types are fully documented (PhpDoc) or explict PHP types are used (type hints, return types, typed properties)

I also fixed some trivial issues reported by my inspection profile.

--

Actions needed:

- [ ] Please check my inline comments. Some of them might be just information, some require an action. You can use `TODO PHP8-REVIEW` when searching.

Best regards,
@mjansenDatabay

### CODE REVIEW

File: Modules/MediaPool/classes/class.ilMediaPoolPage.php [hunk 1]
```diff
@@ -25,15 +25,15 @@ public function getParentType() : string
         return "mep";
     }
 
-    public static function deleteAllPagesOfMediaPool($a_media_pool_id)
+    public static function deleteAllPagesOfMediaPool($a_media_pool_id) : void// TODO PHP8-REVIEW Type hint missing
```
Comment (mjansenDatabay) [inline Modules/MediaPool/classes/class.ilMediaPoolPage.php:28]: Type hint missing

File: Modules/MediaPool/classes/class.ilMediaPoolPage.php [hunk 2]
```diff
@@ -25,15 +25,15 @@ public function getParentType() : string
...
     {
         // @todo deletion process of snippets
     }
     
     /**
      * Checks whether a page with given title exists
      */
-    public static function exists($a_media_pool_id, $a_title)
+    public static function exists($a_media_pool_id, $a_title) : void// TODO PHP8-REVIEW Type hints missing
```
Comment (mjansenDatabay) [inline Modules/MediaPool/classes/class.ilMediaPoolPage.php:36]: Type hints missing

**Comment by alex40724** [comment]
Thanks. I will take care of the todos.

**Comment by alex40724** [comment]
Done.

**Pull request merged**